instance/models/mixins/openedx_database.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Open edX instance database mixin
+"""
+import string
+
+from django.conf import settings
+from django.db.backends.utils import truncate_name
+from django.template import loader
+from django.utils.crypto import get_random_string
+
+from .database import MySQLInstanceMixin, MongoDBInstanceMixin, SwiftContainerInstanceMixin
+
+
+# pylint: disable=too-many-instance-attributes
+class OpenEdXDatabasesMixin(MySQLInstanceMixin, MongoDBInstanceMixin, SwiftContainerInstanceMixin):
```

Comment:
@bradenmacdonald I would be in favor of splitting up, the two are handled differently, even if they both persist data. It would also help to remove the pylint warning here, too.